solidity/hardhat.config.ts

```diff
@@ -19,12 +28,15 @@ usePlugin("buidler-typechain");
 // Go to https://buidler.dev/config/ to learn more
 module.exports = {
   // This is a sample solc configuration that specifies which version of solc to use
-  solc: {
+  solidity: {
     version: "0.6.6",
-    optimizer: { enabled: true, runs: 200000 }
```

Comment:
Might want to look into what different runs numbers do to the gas costs. IIRC the higher the better. Just costs more to deploy the contract